interview-questions.md

```diff
@@ -7,48 +7,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
+  Your answer: It is a snapshot of the object in a class
 
-  Researched answer:
+  Researched answer: The instance variable is an object in a class with a local scope. 
 
 
 
 2. What is a block in Ruby?
 
-  Your answer:
+  Your answer: I assumed it was a block of code in Ruby. 
 
-  Researched answer:
+  Researched answer: A ruby block acts similarly to a method but without a name, which offers liberties that js methods can't offer.
+  " a block is an anonymous piece of code, it can accept input in form of arguments (if it needs any), and it will return a value, but it does not have a name."
 
 
 
 3. Ruby has an implicit return. What does that mean?
 
-  Your answer:
+  Your answer: I do not know.
```

Comment:
Like mentioned above, instead of just leaving it at "I do not know" I'd want to respond with "Not sure! Heres how I'd find out.."